department-of-veterans-affairs/va.gov-cms
# VACMS-16335 Add VAMC System police contact migration

## Description

Relates to #16335

## Testing done


## Screenshots


## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As an admin
1. Go to the [VA Police contact migration](https://pr16430-n3abk8imoxaayufgthipnmezjrkymup5.ci.cms.va.gov/admin/structure/migrate/manage/vamc/migrations)
   - [x] Validate the the row for "Node - VAMC System Police contact" shows 170 total items (the number of data  rows in the CSV)
 
2. Click the Execute button for "vamc_system_police_contact" and Execute the migration with no additional options.
   - [x] Validate that 170 items were processed,  Should show 139 created... which is a like it only updated, but it can't tell.
   
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/6cda866e-6944-42d1-b927-e823d459d9e3)

3. Go to [the migrate messages list](https://pr16430-n3abk8imoxaayufgthipnmezjrkymup5.ci.cms.va.gov/admin/structure/migrate/manage/vamc/migrations/vamc_system_police_contact/messages)
   - [x] validate there are no messages for this migration.
4. Go to the [content list filtered by va police](https://pr16430-n3abk8imoxaayufgthipnmezjrkymup5.ci.cms.va.gov/admin/content?title=&type=vamc_system_va_police&moderation_state=All&owner=All&order=changed&sort=asc) 
5. Toggle the updated date sort back and forth.
   - [x] validate the first item in the list and the last were both updated by the migration.  (times may be a few minutes apart)
6. Go to [content with extension](https://pr16430-n3abk8imoxaayufgthipnmezjrkymup5.ci.cms.va.gov/altoona-health-care/va-police)
   - [x] validate that it has both a phone number and extension
   - [x] Validate it has a revision log message by cms migrator that shows the update of contact info.
7. Go to [content with no extension](https://pr16430-n3abk8imoxaayufgthipnmezjrkymup5.ci.cms.va.gov/amarillo-health-care/va-police) 
   - [x]  validate that it has both a phone number but no extension

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
